docs/tools/aleph-client/documentation.md

```diff
@@ -0,0 +1,942 @@
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `unpersist`: Stop a persistent virtual machine by...
+* `update`: Update the code of an existing program
+* `upload`: Register a program to run on aleph.im.
+
+### `program unpersist`
+
+Stop a persistent virtual machine by making it non-persistent
+
+**Usage**:
+
+```console
+$ aleph program unpersist [OPTIONS] ITEM_HASH
+```
+
+**Arguments**:
+
+* `ITEM_HASH`: [required]
+
+**Options**:
+
+* `--private-key TEXT`
+* `--private-key-file PATH`: [default: /home/mhh/.aleph-im/private-keys/ethereum.key]
+* `--debug / --no-debug`: [default: no-debug]
+* `--help`: Show this message and exit.
+
+### `program update`
+
+Update the code of an existing program
+
+**Usage**:
+
+```console
+$ aleph program update [OPTIONS] ITEM_HASH PATH
+```
+
+**Arguments**:
+
+* `ITEM_HASH`: [required]
+* `PATH`: [required]
+
+**Options**:
+
+* `--private-key TEXT`
+* `--private-key-file PATH`: [default: /home/mhh/.aleph-im/private-keys/ethereum.key]
+* `--print-message / --no-print-message`: [default: print-message]
+* `--debug / --no-debug`: [default: no-debug]
+* `--help`: Show this message and exit.
+
+### `program upload`
+
+Register a program to run on aleph.im. For more information, see https://docs.aleph.im/computing/
+
+**Usage**:
+
+```console
+$ aleph program upload [OPTIONS] PATH ENTRYPOINT
+```
+
+**Arguments**:
+
+* `PATH`: Path to your source code  [required]
+* `ENTRYPOINT`: Your program entrypoint  [required]
+
+**Options**:
+
+* `--channel TEXT`: Aleph.im network channel where the message is located
+* `--memory INTEGER`: Maximum memory allocation on vm in MiB  [default: 256]
+* `--vcpus INTEGER`: Number of virtual cpus to allocate.  [default: 1]
+* `--timeout-seconds FLOAT`: If vm is not called after [timeout_seconds] it will shutdown  [default: 30.0]
+* `--private-key TEXT`: Your private key. Cannot be used with --private-key-file
+* `--private-key-file PATH`: Path to your private key file  [default: /home/mhh/.aleph-im/private-keys/ethereum.key]
+* `--print-messages / --no-print-messages`: [default: no-print-messages]
+* `--print-code-message / --no-print-code-message`: [default: no-print-code-message]
+* `--print-program-message / --no-print-program-message`: [default: no-print-program-message]
+* `--runtime TEXT`: Hash of the runtime to use for your program. Defaults to aleph debian with Python3.8 and node. You can also create your own runtime and pin it
+* `--beta / --no-beta`: If true, you will be prompted to add message subscriptions to your program  [default: no-beta]
+* `--debug / --no-debug`: [default: no-debug]
+* `--persistent / --no-persistent`: [default: no-persistent]
+* `--persistent-volume TEXT`: Persistent volumes are allocated on the host machine and are not deleted when the VM is stopped.
+
+Requires at least a "mount" and "size_mib". For more info, see the docs: https://docs.aleph.im/computing/volumes/persistent/
```

Comment:
Could this be clickable?
by doing something like this:
[docs](https://docs.aleph.im/computing/volumes/persistent/)